src/internal/connector/graph_connector.go

```diff
@@ -79,26 +81,26 @@ func NewGraphConnector(ctx context.Context, acct account.Account, r resource) (*
 
 	gc.Service, err = gc.createService()
 	if err != nil {
-		return nil, errors.Wrap(err, "creating service connection")
+		return nil, clues.Wrap(err, "creating service connection").WithClues(ctx)
 	}
 
 	gc.Owners, err = api.NewClient(m365)
 	if err != nil {
-		return nil, errors.Wrap(err, "creating api client")
+		return nil, clues.Wrap(err, "creating api client").WithClues(ctx)
 	}
 
 	// TODO(ashmrtn): When selectors only encapsulate a single resource owner that
 	// is not a wildcard don't populate users or sites when making the connector.
 	// For now this keeps things functioning if callers do pass in a selector like
 	// "*" instead of.
 	if r == AllResources || r == Users {
-		if err = gc.setTenantUsers(ctx); err != nil {
+		if err = gc.setTenantUsers(ctx, errs); err != nil {
```

Comment:
Is there a particular reason why these errors aren't wrapped with clues?

Reply:
As in, why do we use `errors.Wrap()` instead of `clues.Wrap()`?  Clues is interchangeable with errors, so it won't matter if you wrap using one or the other (unless you need the other clues interface to add additional structured data, too).